src/test/java/org/javarosa/core/model/instance/geojson/GeoJsonExternalInstanceTest.java

```diff
@@ -94,6 +94,12 @@ public void parse_addsAllOtherPropertiesAsChildren() throws IOException {
         assertThat(featureCollection.getChildAt(1).getChild("special-property", 0).getValue().getValue(), is("special value"));
     }
 
+    @Test
+    public void parse_addsFeaturesWithNoProperties() throws IOException {
+        TreeElement featureCollection = GeoJsonExternalInstance.parse("id", r("feature-collection-no-properties.geojson").toString());
+        assertThat(featureCollection.getChildAt(0).getNumChildren(), is(1));
```

Comment:
Does this result in a crash in Collect if the name/label is missing or is there a graceful error?

Reply:
The error is caught but I wouldn't call it graceful:

![device-2022-03-10-100306](https://user-images.githubusercontent.com/967540/157726896-eba01ed4-77cf-425c-96f5-097e8a1be396.png)

Let me think about improving that.